# Bar30SoundSpeed.msg
# - Reads T/D data from MOOS-driven pressure sensor
Header header
float32 water_temp      # Water temperature (degC)
float32 temp_sigma      # Temperature Sigma (degC)
float32 pressure        # Pressure (kPa)
float32 pressure_sigma  # Pressure Sigma (kPa)
float32 sound_speed     # Reported Sound Speed (m/s)
